src/scss/base/_typography.scss

```diff
@@ -18,6 +18,36 @@
 //
 // ******
 
+/**
+ * Typography preloading styles
+ */
+
+.is-fontsLoading--base {
+
+    body,
+    p {
+        font-family: $font-family-preload !important;
+    }
+}
+
+.is-fontsLoading--heading {
```

Comment:
Can you not add these styles into the existing section below using something like `.is-fontsLoading--heading &`? Otherwise, the list of selectors will have to be maintained in two places.

Reply:
🤦‍♂️